src/tuntap.cc

```diff
@@ -140,10 +140,7 @@ bool Tuntap::construct(Handle<Object> main_obj, std::string &error) {
 	else if(this->mode == MODE_TAP)
 		ifr.ifr_flags |= IFF_TAP;
 	
-	if(this->itf_name.size() > 0)
-		MK_IOCTL(this->fd, TUNSETIFF, &ifr)
-	
-	MK_IOCTL(this->fd, TUNGETIFF, &ifr)
+	MK_IOCTL(this->fd, TUNSETIFF, &ifr)
```

Comment:
Well, I didn't know TUNSETIFF would also change the name inside the structure...
The kernel documentation is sadly not that easy to find & read :cry: 
Thanks!